content/800-data-platform/050-data-proxy/index.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Data Proxy'
+metaTitle: 'Data Proxy'
+metaDescription: 'Data Proxy in the Prisma Data Platform provides database connection management and pooling, load balancing, scaling, and fault tolerance features so that you can manage and scale database connections for applications that use Prisma ORM. With Data Proxy, your application can scale and maintain predictable database performance as it also limits the total number of database connections.'
```

Comment:
```suggestion
metaDescription: 'Data Proxy in the Prisma Data Platform provides database connection management and pooling, load balancing, scaling, and fault tolerance features so that you can manage and scale database connections for applications that use Prisma ORM. With Data Proxy, your application can scale and maintain predictable database performance because it also limits the total number of database connections.'
```

[Avoiding "as"](https://www.prisma.io/docs/about/prisma-docs/style-guide/word-choice#avoid-ambiguous-english-words)